src/website/app/App.js

```diff
@@ -119,27 +119,13 @@ class App extends Component<Props> {
     }
   }
 
-  componentDidUpdate(prevProps) {
-    if (canUseDOM && this.props.location !== prevProps.location) {
-      global.window.scrollTo(0, 0);
```

Comment:
Without this one, non-Chrome browsers remembered the scroll position across page-to-page navigation.